cider-dap/src/error.rs

```diff
@@ -0,0 +1,15 @@
+#[allow(dead_code)] //remove this later
+#[derive(thiserror::Error, Debug)]
+pub enum MyAdapterError {
+    #[error("Unhandled command")]
```

Comment:
minor note, please put a space between the enum variants to make it easier to
read. You can also add some doc strings as well, if desired.